jira/client.py

```diff
@@ -1502,7 +1502,7 @@ def create_issue(
         p = data["fields"]["issuetype"]
         if isinstance(p, int):
             data["fields"]["issuetype"] = {"id": p}
-        if isinstance(p, (str, int)):
+        if isinstance(p, str):
```

Comment:
should this be an elif?

Reply:
I reckon it could be an elif. Thanks for the headsup.

---
<sub>Jannik Meinecke (<jannik.meinecke@mercedes-benz.com>) on behalf of MBition GmbH.
[Provider Information](https://github.com/mercedes-benz/foss/blob/master/PROVIDER_INFORMATION.md)</sub>